### DISCUSSION TITLE
MappedSubresource.AsSpan<T> length parameter

### DISCUSSION THREAD

**Discussion by Kharzette** [amerkoleci/Vortice.Windows#288] [Ideas]
Length sort of implies that it might want a count of T for the size of the span, but the code is expecting byte size.

However I've been having a lot of trouble with dotnet 6 sizeof getting my struct sizes wrong, so maybe byte size is better?

If it sticks with byte size, maybe change the name of length to byteSize or sizeInBytes.

**Comment by amerkoleci** [comment]
Thanks,
This commit fixes this one: https://github.com/amerkoleci/Vortice.Windows/commit/38f091a1d4ff53c3d61dfee663a55a3e1db863ea
